test/dao/network-fee-withdraw.ts

```diff
@@ -68,4 +68,14 @@ describe('DAO Network Fee Withdraw Tests', () => {
     await expect(ssvNetworkContract.connect(helpers.DB.owners[3]).withdrawNetworkEarnings(amount
     )).to.be.revertedWith('Ownable: caller is not the owner');
   });
+
+  it('Withdraw network earnings sequentially when not enough balance reverts "InsufficientBalance"', async () => {
+    const amount = await ssvViews.getNetworkEarnings() / 2;
+
+    await ssvNetworkContract.withdrawNetworkEarnings(amount);
+    await ssvNetworkContract.withdrawNetworkEarnings(amount);
+
+    await expect(ssvNetworkContract.withdrawNetworkEarnings(amount
+      )).to.be.revertedWithCustomError(ssvNetworkContract, 'InsufficientBalance');
+  });
```

Comment:
Please for these tests and all other tests where we do any withdraw lets add get assertions.
so in this particular test we do 3 withdrawls and one of them fails
After each one please do a getNetworkBalance and assert the number is correct.

Same for operator withdraw we should do a getOperatorEarnings and assert the number that its correct.